### PR TITLE
[DATA UPDATA] migration script to fix templates linking to removed desks

### PR DESCRIPTION
**Pull request by jerome-poisson**
SDCP-488

### CODE REVIEW

File: server/data_updates/00029_20210315-144037_content_templates.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+
+class DataUpdate(BaseDataUpdate):
+
+    resource = "content_templates"
+    desks_field = "template_desks"
+
+    def forwards(self, mongodb_collection, mongodb_database):
+        """Check templates to remove references to deleted desks (SDCP-488)"""
+        vocabularies_service = get_resource_service(self.resource)
```
Comment (petrjasek) [inline server/data_updates/00029_20210315-144037_content_templates.py]: this should be rather templates_service

File: server/data_updates/00029_20210315-144037_content_templates.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        desks_service = get_resource_service("desks")
+        templates_collection = mongodb_database[self.resource]
```
Comment (petrjasek) [inline server/data_updates/00029_20210315-144037_content_templates.py]: this is already `mongodb_collection` argument

File: server/data_updates/00029_20210315-144037_content_templates.py (hunk 1 above)
Comment (jerome-poisson) [reply server/data_updates/00029_20210315-144037_content_templates.py]: :+1:

File: server/data_updates/00029_20210315-144037_content_templates.py (hunk 2 above)
Comment (jerome-poisson) [reply server/data_updates/00029_20210315-144037_content_templates.py]: oh right, I didn't even realised it was linked to resource, ok fixing it.

**Comment by jerome-poisson** [comment]
Note: I've rebased to `master` branch.

**Review by petrjasek — APPROVED** [review]

**Pull request merged**